tests/coffee_script/CoffeeLintBearTest.py

```diff
@@ -265,6 +265,10 @@ class theGangster
 var test
 """
 
+long_line_length_file = """
```

Comment:
Try replacing this with some meaningful code that actually works :)

Reply:
honestly, does it matter?